
From: janitor@sternwelten.at

 Removes definition of MS_TO_HZ() in favor of msecs_to_jiffies().

Signed-off-by: Nishanth Aravamudan <nacc@us.ibm.com>
Signed-off-by: Maximilian Attems <janitor@sternwelten.at>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/video/aty/radeonfb.h |    2 --
 1 files changed, 2 deletions(-)

diff -puN drivers/video/aty/radeonfb.h~video-radeonfb-remove-ms_to_hz drivers/video/aty/radeonfb.h
--- 25/drivers/video/aty/radeonfb.h~video-radeonfb-remove-ms_to_hz	Thu Sep 23 15:52:02 2004
+++ 25-akpm/drivers/video/aty/radeonfb.h	Thu Sep 23 15:52:02 2004
@@ -425,8 +425,6 @@ static inline u32 _INPLL(struct radeonfb
 		spin_unlock_irqrestore(&rinfo->reg_lock, flags); 	\
 	} while (0)
 
-#define MS_TO_HZ(ms)       ((ms * HZ + 999) / 1000)
-
 #define BIOS_IN8(v)  	(readb(rinfo->bios_seg + (v)))
 #define BIOS_IN16(v) 	(readb(rinfo->bios_seg + (v)) | \
 			  (readb(rinfo->bios_seg + (v) + 1) << 8))
_
